lookout/style/format/analyzer.py

```diff
@@ -135,7 +142,7 @@ def train(cls, ptr: ReferencePointer, config: Mapping[str, Any], data_request_st
                  "max_features": Categorical([None, "auto"]),
                  "min_samples_split": Integer(2, 20),
                  "min_samples_leaf": Integer(1, 20)},
-                n_jobs=-1,
+                n_jobs=config["n_jobs"],
```

Comment:
+1
However I fear that when we need it (ie big repos) it will become so slow that it'll also be unbearable. We could probably experiment with subsambling if we want to find another way to mitigate both memory and speed issues.

Reply:
at least it could solve the problem for local experiments when RAM is quite limited + this argument allows to reduce number of iterations https://github.com/src-d/style-analyzer/blob/688d3468a0da4b4178f49e44c43ac5b222d460ec/lookout/style/format/analyzer.py#L146

Reply:
true. Btw you can rebase #153 if you want to reduce memory impact, it helps a lot already :)